standard.md

```diff
@@ -0,0 +1,272 @@
+Any change in `id` is **Non-Breaking**
+
+### `defaultContentType`
+
+Change in `defaultContentType` is **Breaking**
+
+### `info`
+
+**Breaking:**
+
+1. Change in `version`
+1. Change in `termsOfService`
+1. Change in `license` field
+
+**Non-Breaking:**
+
+1. Change in `title`
+1. Change in `description`
+1. Change in `contact` field
+
+### `servers`
+
+**Breaking:**
+
+1. Removal of a server(equivalent to change in server name)
+1. Change in `url`
+1. Change in `protocol`
+1. Change in `protocolVersion`
+1. Change in `default` (`variables`)
+1. Removal in `enum` (`variables`)
+1. Change in `security`
+
+**Non-Breaking:**
+
+1. Server being added
+1. Change in `description`
+1. Change in `description` (`variables`)
+1. Change in `examples` (`variables`)
+1. Addition in `enum` (`variables`)
+
+### `channels`
+
+**Breaking:**
+
+1.  Removal of channel
+1.  Removal of `subscribe` or `publish` field
+1.  Change in value of `$ref`
+
+**Non-Breaking:**
+
+1. Addition of a channel
+1. Change in `description`
+1. Addition of `subscribe` or `publish` field
+
+#### `subscribe` | `publish`
+
+**Breaking:**
+
+1. Change in `operationId`
+1. Change in `traits`
+
+**Non-Breaking:**
+
+1. Change in `summary`
+1. Change in `description`
+1. Change in `externalDocs`
+1. Change in `tags`
+
+##### `message`
+
+**Breaking:**
+
+1. Change in `contentType`
+1. Change in `name`
+1. Change in `payload`
+1. Change in `traits`
+1. Change in `headers`
+
+**Non-Breaking:**
+
+1. Change in `summary`
+1. Change in `description`
+1. Change in `examples`
+1. Change in `title`
+1. Change in `tags`
+1. Change in `externalDocs`
+1. Change in `correlationId`
+
+#### `parameters`
+
+**Breaking:**
+
+1. Change in `schema`
+1. Change in `location`
+1. Change in value of `$ref`
+
+**Non-Breaking:**
+
+1. Change in `description`
+
+### `components`
+
+Any change in `components` is **non-breaking**.
+
+**Reason:** The parser will inline the `$ref` during the parsing phase.
+
+### `tags`
+
+Any change in `tags` is **non-breaking**.
+
+### `externalDocs`
+
+Any change in `externalDocs` is **non-breaking**.
+
+## Summary
+
+### Breaking changes
+
+1. Change in `major` version in `asycnapi`
+1. Change in `defaultContentType` field
+1. Change in `version` in `info`
+1. Change in `termsOfService` in `info`
+1. Change in `license` field in `info`
+1. Removal of a server(equivalent to change in server name)
+1. Change in `url` in `servers`
+1. Change in `protocol` in `servers`
+1. Change in `protocolVersion` in `servers`
+1. Change in `default` (`variables`) in `servers`
+1. Removal in `enum` (`variables`) in `servers`
+1. Change in `security` in `servers`
+1.  Removal of channel
+1.  Removal of `subscribe` or `publish` field in `channels`
+1.  Change in value of `$ref` in `channels`
+1. Change in `operationId` in `subscribe`|`publish`
+1. Change in `traits` in `subscribe`|`publish`
+1. Change in `contentType` in `message`
+1. Change in `name` in `message`
+1. Change in `payload` in `message`
```

Comment:
To me this cannot be stated, what if you change the description of a property when using JSON Schema? 🤔 That should not be considered a breaking change. 

What's the reason behind assuming changes to payload is breaking? 

I could see that this is just for more easily assume it, as you then would need to diff JSON Schema as well, which is gonna take more time 😅

Reply:
I completely overlooked that point, but you are right, changing the `description` and such in message payload is not a breaking change. Will update the document :)

Reply:
I think we could try to define a basic minimum of non-breaking changes in JSON Schema in payload and threat the rest as breaking, or? later it could evolve into something separate? or we are complete excluding `payload` content from Diff because AsyncAPI allows also payloads with Avro and other formats? This way we would always treat change in payload as breaking, clearly documented and explained of course

Reply:
> I think we could try to define a basic minimum of non-breaking changes in JSON Schema in payload and threat the rest as breaking, or?

I thought about this. Like, we can define a set of properties as non-breaking and mark others as breaking. But I'm not too sure if we should do that or not.

> we are complete excluding payload content from Diff because AsyncAPI allows also payloads with Avro and other formats?

I don't think so. Like @jonaslagoni said, changing the description will throw us a "breaking" change error where it is not.

But then again, do we want to go through the whole JSON Schema and mark everything as breaking/non-breaking?
If that's the case, we can go with the first approach. Only put "non-breaking" properties in the standard, and treat everything else as "breaking"

Reply:
both solutions are bad and not something user can rely on. Wouldn't it make sense to have 3rd category like, `uncategorized`? or `unknown`?

Reply:
Yeah, I think having an `unknown` or `uncategorized` category will be good. 

So, what message do you think should `uncategorized` convey? I'm thinking of something like "these changes are uncategorized, the user should handle these themselves".

Reply:
So, I have made some changes to the standard. Now it includes `unclassfied` changes as well. Right now, the message `headers` and `payload` are being marked as `unclassified`.